policyengine_us/reforms/congress/golden/fisc_act.py

```diff
@@ -0,0 +1,98 @@
+from policyengine_us.model_api import *
+from policyengine_core.periods import period as period_
+from policyengine_core.periods import instant
+
+
+def create_fisc_act() -> Reform:
+    class family_income_supplement_credit(Variable):
+        value_type = float
+        entity = TaxUnit
+        label = "FISC Act family income supplement"
+        unit = USD
+        definition_period = YEAR
+        reference = "https://golden.house.gov/sites/evo-subsites/golden.house.gov/files/evo-media-document/GoldenFISC.pdf"
+
+        def formula(tax_unit, period, parameters):
+
+            # Calcualte the base amount
+            person = tax_unit.members
+            is_pregnant = person("is_pregnant", period)
+            is_dependent = person("is_tax_unit_dependent", period)
+            age = person("age", period)
+            p = parameters(
+                period
+            ).gov.contrib.congress.golden.fisc_act.family_income_supplement
+            eligible_dependent = (age < p.child_age_threshold) & is_dependent
+            pergnant_amount = p.amount.pregnant * is_pregnant
+            dependent_amount = p.amount.base.calc(age) * eligible_dependent
+            total_base_amount = (
+                tax_unit.sum(pergnant_amount + dependent_amount)
+                * MONTHS_IN_YEAR
```

Comment:
this doesnt really work for pregnancy since only available from 20 weeks (4 monthly payments in general)